### ISSUE TITLE
Unable to synchronize to Github

### ISSUE THREAD

**Issue by sanjayabc1234** [benweet/stackedit#1807]
Hello Team,

I have a private repository. I have been using Stackedit (online version) for long time. Recently I am not able to open from Github or create any new file on Github using StackEdit application. I get following error for the rest api in my browser network console: 401 error.

API: /repos/{owner}/{repo}/contents/{path}

```json
{
    "message": "Bad credentials",
    "documentation_url": "https://docs.github.com/rest"
}
```
I revoked the access from Githhub and authorised it again. But this isn't resolved the issue.

Thank you,

**Comment by edmcman** [comment]
https://github.com/benweet/stackedit/pull/1724

**Comment by mafgwo** [comment]
My fork repo has fixed this bug.
StackEdit+ : https://stackedit.net
open source: https://github.com/mafgwo/stackedit-plus

**Comment by benweet** [comment]
Should be fixed with v5.15.0.
